data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		outfits "Afterburner (Stellar Class)"
+	on offer
+		conversation
+			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life signs aboard, most of which read as likely intelligent, with many thousands more minor life signs alongside them.`
+			`	Some of these minor life signs are flora, like what you saw in the massive dome atop the craft, as well as a wide variety of fauna. The rest, though, seem to be large orbs found throughout the craft, mostly in large groupings. A crew member identifies the orbs as eggs - undoubtedly Korath eggs.`
```

Comment:
I think this can trigger even if the player doesn't really know who the Korath are (no first contact, not knowing how lizard-like they are).

Reply:
Is there a first contact for the exiles, I know there is efreti ones for each of their planets, but I don't remember there being any exile one?

Reply:
I'm not sure. It just felt weird to talk about Korath eggs without making sure the player has prior knowledge.

Reply:
Not yet.  There used to be, then it was removed when the Efreti were added to the game, as MZ was unsatisfied with how it didn't go anywhere.  #8752 seeks to restore and expand it.

Reply:
Alright, so just a case of once that is added, add it as a requirement, I just would assume if you are able to take out an A'awoj, you have boarded Korath vessals before currently :P